packages/docusaurus-theme-classic/src/theme/Navbar/styles.css

```diff
@@ -26,6 +26,24 @@
   -webkit-tap-highlight-color: transparent;
 }
 
+.toggle {
```

Comment:
`.toggle` is an extremely common class name and we either namespace it with if possible (with prefix or CSS modules). Since we have CSS modules let's use it.

Reply:
agree

Reply:
Updated to use CSS Modules 🎃